skimage/restoration/deconvolution.py

```diff
@@ -8,62 +8,63 @@
 
 
 def wiener(image, psf, balance, reg=None, is_real=True, clip=True):
-    r"""Wiener-Hunt deconvolution
+    r"""Restore image using Wiener-Hunt deconvolution.
 
-    Return the deconvolution with a Wiener-Hunt approach (i.e. with
-    Fourier diagonalisation).
+    Wiener-Hunt deconvolution is a restoration method which follows a Bayesian
```

Comment:
It should be an en dash between 'Wiener' and 'Hunt' because Wiener is one person and Hunt another one.